styles/variables-silicon.less

```diff
@@ -470,7 +470,7 @@
 
 // WindDirectionControl
 // ---------------------------------------
-@agate-winddirectioncontrol-icon-font-size: 210px;
-@agate-winddirectioncontrol-icon-height: 210px;
-@agate-winddirectioncontrol-icon-line-height: 210px;
-@agate-winddirectioncontrol-icon-width: 210px;
+@agate-winddirectioncontrol-icon-font-size: 240px;
+@agate-winddirectioncontrol-icon-height: 240px;
+@agate-winddirectioncontrol-icon-line-height: 240px;
+@agate-winddirectioncontrol-icon-width: 240px;
```

Comment:
Where comes these numbers from?

Reply:
It's double the size of the icon element in the GUI, so that the icon looks the same size in the sampler as it does in the GUI.